com.avaloq.tools.ddk.xtext.builder/src/com/avaloq/tools/ddk/xtext/builder/MonitoredClusteringBuilderState.java

```diff
@@ -794,15 +794,12 @@ protected boolean recordDeltaAsNew(final Delta newDelta) {
    *          The new index
    * @param monitor
    *          The progress monitor used for user feedback
-   * @return a List with the list of loaded resources {@link URI} in the first position and a list of {@link URI}s of resources that could not be loaded in the
-   *         second position.
+   * @return a List with the list of loaded resources {@link URI}

```

Comment:
comment should be adjusted. It is no longer list of lists

Reply:
Done